kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/CertManager.java

```diff
@@ -45,7 +45,7 @@ public CertManager() throws IOException {
     public void deploy() {
         LOGGER.info("Deploy cert manager in {} namespace", Constants.CERT_MANAGER_NAMESPACE);
         if (kubeClient().getNamespace(Constants.CERT_MANAGER_NAMESPACE) != null
-            || Environment.CERT_MANAGER_INSTALLED.equalsIgnoreCase("true")) {
+                || Environment.CERT_MANAGER_INSTALLED.equalsIgnoreCase("true")) {
```

Comment:
`java.lang.Boolean#parseBoolean()` is the prefered way of turning a string into a truth value.